server/controllers/logController.js

```diff
@@ -0,0 +1,30 @@
+//
+// Controller Log
+//
+module.exports = function initLogController(libs, managers, controllers) {
+
+  //
+  // Get libs and header,
+  // launch function startLog from managers
+  // and return the logger.
+  // The header is composed of
+  // userId and moduleName.
+  //
+  controllers.launchLog = function (libs, header) {
+    return managers.launchLog(libs, header);
+  };
+
+  //
+  // Launch getLogsById from managers and return the result
+  //
+  controllers.getLogsById = function (userId) {
+    return managers.getLogsById(userId);
+  };
+
+  //
+  // Launch getLimitedLogsById from managers and return the result
+  //
+  controllers.getLimitedLogsById = function (userId, limit) {
+    return managers.getLimitedLogsById(userId, limit);
+  };
+};
```

Comment:
@michau-k @vuongc @StevenIndigo 
I still need a confirmation for that from @BernardJeremy for some similar cases, but, for now, and from my point of view, the controllers should only be the entry point that will initialize all the underlying functionalities. Because of that, the actual code inside the log controller should be moved to the log manager. In the sense of, the functionalities should not be callable from the controller, by from the managers.